frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx

```diff
@@ -0,0 +1,200 @@
+      expect(depositBtn).toBeInTheDocument();
+      expect(withdrawBtn).toBeInTheDocument();
+      expect(historyBtn).toBeInTheDocument();
+      expect(editBtn).toBeInTheDocument();
+      expect(bountiesBtn).toBeInTheDocument();
+    });
+  });
+
+  it('should disable edit and add user button if user is not admin', async () => {
+    jest.spyOn(mainStore, 'getPersonAssignedBounties').mockReturnValue(Promise.resolve([]));
+    jest.spyOn(mainStore, 'pollOrgBudgetInvoices').mockReturnValue(Promise.resolve([]));
+    jest.spyOn(mainStore, 'organizationInvoiceCount').mockReturnValue(Promise.resolve(0));
+    jest.spyOn(mainStore, 'getOrganizationUsers').mockReturnValue(Promise.resolve([person]));
+    jest.spyOn(mainStore, 'getUserRoles').mockReturnValue(Promise.resolve([]));
+    jest
+      .spyOn(mainStore, 'getOrganizationBudget')
+      .mockReturnValue(Promise.resolve({ total_budget: 10000 }));
+    jest.spyOn(mainStore, 'getPaymentHistories').mockReturnValue(Promise.resolve([]));
+    const closeFn = jest.fn();
+    const resetOrgFn = jest.fn();
+    const getOrgFn = jest.fn();
+    await act(async () => {
+      const { getByRole } = render(
+        <OrganizationDetails
+          close={closeFn}
+          getOrganizations={getOrgFn}
+          org={organization}
+          resetOrg={resetOrgFn}
+        />
+      );
+
+      const editBtn = getByRole('button', { name: 'Edit' });
+      const addUsersBtn = getByRole('button', { name: 'Add User' });
+
+      expect(editBtn).toBeDisabled();
+      expect(addUsersBtn).toBeDisabled();
+    });
+  });
+
+  it('should disable view bounties button if organization doesnt have any bounty', async () => {
+    jest.spyOn(mainStore, 'getPersonAssignedBounties').mockReturnValue(Promise.resolve([]));
+    jest.spyOn(mainStore, 'pollOrgBudgetInvoices').mockReturnValue(Promise.resolve([]));
+    jest.spyOn(mainStore, 'organizationInvoiceCount').mockReturnValue(Promise.resolve(0));
+    jest.spyOn(mainStore, 'getOrganizationUsers').mockReturnValue(Promise.resolve([person]));
+    jest.spyOn(mainStore, 'getUserRoles').mockReturnValue(Promise.resolve([]));
+    jest
+      .spyOn(mainStore, 'getOrganizationBudget')
+      .mockReturnValue(Promise.resolve({ total_budget: 10000 }));
+    jest.spyOn(mainStore, 'getPaymentHistories').mockReturnValue(Promise.resolve([]));
+    const closeFn = jest.fn();
+    const resetOrgFn = jest.fn();
+    const getOrgFn = jest.fn();
+    await act(async () => {
+      const { getByRole } = render(
+        <OrganizationDetails
+          close={closeFn}
+          getOrganizations={getOrgFn}
+          org={{ ...organization, bounty_count: 0 }}
+          resetOrg={resetOrgFn}
+        />
+      );
+
+      const viewBountiesBtn = getByRole('button', { name: 'View Bounties open_in_new' });
+
+      expect(viewBountiesBtn).toBeDisabled();
+    });
+  });
+
+  it('should disabled hostroy and withdraw button if user is not admin', async () => {
```

Comment:
@wright-eric fix the typo error by changing `hostroy` to `history`, so I can merge.

Reply:
Updated